index.html

```diff
@@ -1,7 +1,9 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
Please add a responsive layout to your project. 
Free CodeCamp's responsive web design course projects are requiring implementing media queries for tablet and mobile view.